coding-agent/fix_issue.py

```
@@ -0,0 +1,163 @@
+import argparse
+import subprocess
+import os
+import re
+import sys
+import time
+
+def run_command(command, cwd=None, capture_output=True, env=None):
+    """Runs a shell command and returns the result."""
+    try:
+        print(f"Running: {command}")
+        result = subprocess.run(
```

Ruff (0.14.5)

`subprocess` call with `shell=True` identified, security issue

(S602)